kds/kds.go

```diff
@@ -700,6 +700,11 @@ func ProductString(product *pb.SevProduct) string {
 	}
 }
 
+// DefaultProductString returns the product string of the default SEV product.
+func DefaultProductString() string {
+	return ProductString(abi.DefaultSevProduct())
+}
+
 // ProductName returns the expected productName extension value for the product associated
 // with an attestation report proto.
 func ProductName(product *pb.SevProduct) string {
```

Comment:
Since product.Stepping is marked as deprecated, shall we replace all its use by machine_stepping? Here and elsewhere

Reply:
I replaced all the uses, but I don't agree with the linter's blocking error here. We should have a period of time where we have both fields so we can more gradually transition.